azext_iot/commands.py

```diff
@@ -188,6 +188,13 @@ def load_command_table(self, _):
             "validate-messages", "iot_central_validate_messages", is_preview=True
         )
 
+    with self.command_group(
+        "iot central app capability-model", command_type=iotcentral_ops
+    ) as cmd_group:
+        cmd_group.command(
+            "get", "iot_central_device_capability_model_get", is_preview=True
```

Comment:
It seems like the whole `az iot central app` command group should be set to is_preview.

Reply:
Monitor-events is not `is_preview`

Reply:
Double checking on this - that is intended right? central app monitor-events is not supposed to be in preview? Please resolve the comment if true.

Reply:
Confirmed with Jack, it is NOT in preview